docs/docs/spark-writes.md

```diff
@@ -332,6 +332,30 @@ The writer must enable the `mergeSchema` option.
 ```scala
 data.writeTo("prod.db.sample").option("mergeSchema","true").append()
 ```
+
+### Writing to Branches
+The branch must exist before performing write. The operation does **not** create the branch if it does not exist.
```

Comment:
Initially, the "Writing to Branches" seemed a first level header (along with "Writing with SQL" and "Writing with Dataframes"), was it intentional?  Do you think we can just add an example using Dataframe here?

My thought is, then we dont have to repeat this block of text, which we will have to do if we downgrade it to subsections within "Writing with SQL" and "Writing with Dataframes".

Reply:
Should we remove these then?

Reply:
Done